osquery/filesystem/filesystem.cpp

```diff
@@ -77,14 +77,18 @@ Status writeTextFile(const fs::path& path,
 
 struct OpenReadableFile {
  public:
-  explicit OpenReadableFile(const fs::path& path) {
+  explicit OpenReadableFile(const fs::path& path, bool is_nonblocking = true) {
```

Comment:
Q: is there precedent for using `is_`? Can we name this `bool blocking = false`?